src/components/Button/Button.tsx

```diff
@@ -17,7 +17,7 @@ export interface DialButtonProps
   > {
   variant?: ButtonVariant;
   textClassName?: string;
-  label?: string;
+  label?: ReactNode;
```

Comment:
Since this change adds support for more than a plain string, should we also update the **tests** and **Storybook stories** to cover the new behavior?